tex/generic/pgf/utilities/pgfkeys.code.tex

```diff
@@ -749,7 +749,7 @@
   {%
     \pgfkeysgetvalue{\pgfkeyscurrentpath/.@args}{\pgfkeys@tempargs}%
     \pgfkeysgetvalue{\pgfkeyscurrentpath/.@body}{\pgfkeys@tempbody}%
-    \def\pgfkeys@marshal{\expandafter\gdef\expandafter\pgfkeys@global@temp\pgfkeys@tempargs}%
+    \def\pgfkeys@marshal{\expandafter\long\expandafter\gdef\expandafter\pgfkeys@global@temp\pgfkeys@tempargs}%
```

Comment:
You don't actually need an `\expandafter` before `\long`.  This would just work fine as well.
```suggestion
    \def\pgfkeys@marshal{\long\expandafter\gdef\expandafter\pgfkeys@global@temp\pgfkeys@tempargs}%
```

Reply:
Ah yes. I might be affected/misled by other two `\expandafter\long` in `pgfkeys.code.tex`.